rmw_opensplice_cpp/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ endif()
 ament_package()
 
 install(
+  DIRECTORY include/
+  DESTINATION include
+)
```

Comment:
Then I should also list `include` as an argument for `ament_export_include_directories(..)`.